fe/fe-core/src/main/java/com/starrocks/catalog/MvRefreshArbiter.java

```diff
@@ -186,23 +187,26 @@ public static MvBaseTableUpdateInfo getMvBaseTableUpdateInfo(MaterializedView mv
             if (baseUpdatedPartitionNames == null) {
                 return null;
             }
-            Map<Table, Column> partitionTableAndColumns = mv.getRefBaseTablePartitionColumns();
-            if (!partitionTableAndColumns.containsKey(baseTable)) {
+            Map<Table, List<Column>> refBaseTablePartitionColumns = mv.getRefBaseTablePartitionColumns();
```

Comment:
only MV on external tables support multi-column partition-by?

Reply:
No, Native List Partition tables can also support multi partition columns which can be base tables of MV too.